indexing-service/src/main/java/org/apache/druid/indexing/common/task/AppenderatorDriverRealtimeIndexTask.java

```diff
@@ -213,6 +215,20 @@ public String getType()
     return "index_realtime_appenderator";
   }
 
+  @JsonIgnore
+  @Nullable
+  @Override
+  public Set<String> getInputSourceTypes()
+  {
+    return null;
```

Comment:
Is this the right answer? Or, should it be `ImmutableSet.of()` so it is non-null? Here and below.